py-polars/polars/dataframe/frame.py

```diff
@@ -1180,6 +1180,18 @@ def dtypes(self) -> list[PolarsDataType]:
         """
         return self._df.dtypes()
 
+    @property
+    def flags(self) -> dict[str, dict[str, bool]]:
+        """
+        Get flags that are set on the DataFrame.
```

Comment:
Can we disambiguate this to "flags set on the columns of this DataFrame". 

A `DataFrame` itself doesn't have any flags as it is a newtype wrapper around `Vec<Series>`.

Reply:
right, thanks!